.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["latest", "stage-2"]
+  "presets": ["env", "stage-2"]
```

Comment:
Should we need to configure `env`? Considering this package only supports Node v4, we could probably do something like [this](https://github.com/kentcdodds/nps-utils/blob/master/.babelrc#L3-L7)